frameworks/person-escort-record/questions/observation-level.yml

```diff
@@ -23,6 +23,9 @@ options:
   -
     label: Constant physical watch
     value: Constant physical watch
+  -
+    label: Standard observation level - no additional observation required
```

Comment:
Can we use an [emdash](thepunctuationguide.com/em-dash.html) instead of the hyphen?

Reply:
Another thought, should this be the first option? Feels like it would keep in a more logical order that way of least risk -> highest risk.

With this change we now go minimal risk -> highest risk -> least risk.